Cargo.toml

```
@@ -13,7 +13,7 @@ repository = "https://github.com/paritytech/cachepot/"
 readme = "README.md"
 categories = ["command-line-utilities", "development-tools::build-utils"]
 keywords = ["ccache", "sccache", "cachepot"]
-edition = "2018"
+edition = "2021"
```

With rust 2021 we could also specify the msrv as part of the package: https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field currently this is `1.56.1`
```suggestion
edition = "2021"
rust-version = "1.56.1"
```

---

For reference, I tried to get this to build with 1.56.0 and it failed with
```
error: package `...` cannot be built because it requires rustc 1.56.1 or newer, while the currently active rustc version is 1.56.0
```
Just as we wanted to happen :rocket: (will propagate to other PRs updating a few crates)